src/Controller/UserController.php

```diff
@@ -107,6 +121,119 @@ public function getManageUsersAction()
         $users = $repository->findAll();        
         $dm->flush();
 
-        return $this->render('rest/users.html.twig', array("userlist" => $users));
+        return $this->render('users.html.twig', array("userlist" => $users));
+    }
+
+    /**
+     * Matches /users/edit/{id}
+     *
+     * @Route("/users/edit/{id}")
+     * @Method("GET")
+     * Method will used for editing the user
+     */
+
+    public function editUserAction(string $id=null, Request $request)
+    {
+        $user = new User();
+        $dm = $this->get('doctrine_mongodb')->getManager();
+        $repository = $dm->getRepository(User::class);
+
+        $educationArray = [            
+            'UG College' => 'UG College',
+            'PG College' => 'PG College',
+            'Masters College' => 'Masters College',
+            'Others College' => 'Others College',
+        ];
+
+        if($id != null) {
+            $users = $repository->findBy(['id' => $id]);
+            $users = $users[0]; 
+        
+            $userEmailString  = implode(",",$users->getUserEmail());
+            $userMobileString = implode(",",$users->getUserMobileNumber());
+            $this->userEduArray = $users->getUserEducation();
+
+            $form = $this->createFormBuilder($user)
+                ->add('userFirstName', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserFirstName()
+                ))
+                ->add('userLastName', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserLastName()
+                ))
+                ->add('userEmail', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userEmailString
+                ))
+                ->add('userMobileNumber', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userMobileString
+                ))
+                ->add('userDateofBirth', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserDateofBirth()
```

Comment:
these are not recommended approaches in symfony.

Reply:
I have modified the form as symfony way  approach.